ember-cli-build.js

```diff
@@ -305,22 +286,22 @@ module.exports = function(options) {
 
     let emberMinBundle = minify(emberProdBundle, 'ember.min');
 
-    trees.push(emberRuntimeBundle, emberProdBundle, emberMinBundle, emberProdTestsBundle, prodTemplateCompiler);
-  } else {
-    let emberTemplateCompilerBundle = new MergeTrees([
-      ...templateCompiler(babelDebugHelpersES5),
-        version,
-        license,
-        loader,
-        nodeModule
-    ]);
+    trees.push(emberRuntimeBundle, emberProdBundle, emberMinBundle, emberProdTestsBundle);
+  }
 
-    emberTemplateCompilerBundle = concat(emberTemplateCompilerBundle, {
-      outputFile: 'ember-template-compiler.js'
-    });
+  let emberTemplateCompilerBundle = new MergeTrees([
+    ...templateCompiler(babelDebugHelpersES5),
```

Comment:
I believe that the spread operator is behind the harmony flag in node 4: http://node.green/#ES2015-syntax-spread-------operator